src/main/resources/application.yml

```diff
@@ -0,0 +1,38 @@
+spring:
+  kafka:
+    bootstrap-servers: localhost:9092
+    consumer:
+      group-id: company-links-consumer
+      auto-offset-reset: earliest
+      key-deserializer: org.apache.kafka.common.serialization.StringDeserializer
+      value-deserializer: org.apache.kafka.common.serialization.StringDeserializer
+
+insolvency:
+  stream:
+    group-id: company-links-consumer
+    topic:
+      main: stream-insolvency-topic
+      retry: stream-insolvency-topic-retry
+      error: stream-insolvency-topic-error
+
+charges:
+  stream:
+    group-id: company-links-consumer
```

Comment:
@zahidm3  Do you agree that we should just use the group id in `spring.kafka.consumer.group-id` rather than here?

Reply:
@dominicst-ch , @adeshpandech , we can keep this config and remove whole of below section from application.yml as we are having multiple consumers in single component and each have different value deserializers. Lets remove the below section instead?

    consumer:
      group-id: insolvency-delta-consumer
      auto-offset-reset: earliest
      key-deserializer: org.apache.kafka.common.serialization.StringDeserializer
      value-deserializer: org.apache.kafka.common.serialization.StringDeserializer

Reply:
thanks, section removed now